### PR TITLE
Update installer-hacks

### PR DESCRIPTION
**Pull request by joseivanlopez**
Add comments to installer-hacks for autoyast2-install.

### CODE REVIEW

File: doc/installer-hacks.md [hunk 1]
```diff
@@ -105,3 +105,17 @@ equivalent, the old code is simply deleted, not commented.
 
 * Code dealing with BIOS-ID changed to blindly assume common values. Original
   code commented and marked with "storage-ng".
+
+## Changes in autoyast2-installation
```
Comment (ancorgs) [inline doc/installer-hacks.md]: I would say autoyast2. That's how the package is called at OBS. autoyast2-installation is just a subpackage and the changes affect the whole OBS package.

File: doc/installer-hacks.md [hunk 2]
```diff
@@ -105,3 +105,17 @@ equivalent, the old code is simply deleted, not commented.
...
+
+* Commented some parts in `AutoInstallRules` to not use old storage lib (commented
+  parts marked with `storage-ng`). This fix problem with intall process (see
```
Comment (ancorgs) [inline doc/installer-hacks.md]: typo: intall

File: doc/installer-hacks.md [hunk 3]
```diff
@@ -105,3 +105,17 @@ equivalent, the old code is simply deleted, not commented.
...
 * Code dealing with BIOS-ID changed to blindly assume common values. Original
   code commented and marked with "storage-ng".
+
+## Changes in autoyast2-installation
+
+* Commented some parts in `AutoInstallRules` to not use old storage lib (commented
+  parts marked with `storage-ng`). This fix problem with intall process (see 
+  [this PBI](https://trello.com/c/qsxBrzIE/499-2-storageng-get-failing-openqa-test-fixed-by-adjusting-autoinstallrules-to-storage-ng)).
+
+* Several tests have been skipped ir order to create the package. Commented tests
```
Comment (ancorgs) [inline doc/installer-hacks.md]: Typo: ir

File: doc/installer-hacks.md [hunk 4]
```diff
@@ -105,3 +105,17 @@ equivalent, the old code is simply deleted, not commented.
...
+  need `yast2-installation`, but it is not possible to provide it due to conflict between `yast2-storage` and `yast2-storage-ng`.
+
+* `yast2-installation` requires `autoyast2-installation` that requires `yast2-update`
+  that requires `yast2-storage`.
+
+* There is a cyclic dependency between `yast2-installation` and `autoyast2-installation`.
```
Comment (ancorgs) [inline doc/installer-hacks.md]: This two last sentences are not separate points, but the explanation of the second one.

Comment (kobliha) [inline doc/installer-hacks.md]: Oops, if we have a cyclic dependency, we should resolve it somehow. Please, report a bug and we can plan for it. Thanks :)

**Comment by ancorgs** [comment]
The failing tests are expected and not related to this PR.

**Pull request merged**